### DISCUSSION TITLE
US based Proxmox hosts for this role?

### DISCUSSION THREAD

**Discussion by niccolox** [lae/ansible-role-proxmox#162] [Q&A]
hey

have you (or any of the users of this role) used it to deploy clusters on US based dedicated server hosts?

I've had servers at OVH in Quebec, but want to stay within US jurisdiction

any suggestions for Proxmox friendly hosters where this role has been tested or used in production?

**Comment by lae** [comment]
@niccolox This is a question better suited for the pve-user mailing list (https://pve.proxmox.com/pipermail/pve-user/). What you want is a dedicated server host that provides private networking + multicast. I personally have only used Proxmox on colocated hardware so can't really give any suggestions.
